File: macos/controllers/profile-view-controllers/MUPlayerViewController.m
//
// MUPlayerViewController.m
//
// Copyright (c) 2013 3James Software.
//

#import "MUPlayerViewController.h"

@implementation MUPlayerViewController

- (instancetype) init
{
  if (!(self = [super initWithNibName: @"MUEditPlayerView" bundle: nil]))
    return nil;
  
  return self;
}

- (void) awakeFromNib
{
  self.view.autoresizingMask = NSViewWidthSizable;
}

@end

File: macos/controllers/profile-view-controllers/MUPlayerViewController.swift
//
//  MUPlayerViewController.swift
//  Koan
//
//  Created by Tyler Berry on 5/10/18.
//  Copyright © 2018 3James Software. All rights reserved.
//

class MUPlayerViewController: MUProfileSubviewController
{
  var player: MUPlayer?
  @IBOutlet var clearTextButton: NSButton?
  
  override var nibName: String { return "MUEditPlayerView" }
  
  override func awakeFromNib()
  {
    view.autoresizingMask = .viewWidthSizable
  }
}
